### PR TITLE
report rendering api

### PR DESCRIPTION
**Pull request by doorooful**
- API endpoint: "/api/filter/report"
- Added sample datat to render html from API request

### CODE REVIEW

**Comment by doorooful** [comment]
API Result on : http://127.0.0.1:5001/api/filter/report

Screenshot 2023-09-15 at 7 43 28 PM

**Comment by doorooful** [comment]
Please download the bootstrap components, unzip, and kindly copy the style files under `dt-demo-be/static/`:
``` example filetree
dt-demo-be
├── app.py
├── README.md
└── modules
└── static
    ├── css
    └── js
└── templates
```
  
Here's bootstrap file for our project:
[bootstrap-5.2.3-dist.zip](https://github.com/doorooful/CapstoneProject/files/12618628/bootstrap-5.2.3-dist.zip)

File: dt-demo-be/app.py [hunk 1]
```diff
@@ -35,6 +36,27 @@ def filter():
...
+    report_data_sample = {
+        'title': "Report",
+        'title_fileName': "demo.txt",
+        'overview_rate': "10%",
+        'overview_typo': 5,
+        'overview_slang': 3,
+        'overview_pdd': 2,
+        'overview_dup': 4,
+        'overview_char': 6,
+        'overview_all': 20,
```
Comment (yoouung) [inline dt-demo-be/app.py:52]: Please set the calculation of the total amount of filtered data

**Comment by doorooful** [comment]
Please find the recent changes !!

**Review by yoouung — APPROVED** [review]
LGTM!

**Comment by doorooful** [comment]
```plantuml
@startuml
title Report processing

-> App: GET("api/filter/report")
App -> App: createSampleData()
App -> Receiver: return render_template_in_HTML
@enduml
```

**Pull request merged**